### DISCUSSION TITLE
how to use Newtek TC2 mixer with rtmp

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#5184] [FAQs] [answered]
I'm planning to use Newtek TC2 as the main mixer. How should I send the signal to Ant Media Server as RTMP or is it better to send it  as " NDI webcam" with broadcast page on Ant Media Server?

**Comment by Mohit-3196** [comment, marked as answer]
Teradek Vidiu Pro can be used to achieve this and send it to AMS with RTMP and it is compatible with Ant Media Server.
There's a detailed instruction on using Teradek with AMS that can be referred to here https://antmedia.io/docs/guides/publish-live-stream/rtmp/publish-with-teradek/

**Comment by mekya** [comment]
[Some versions of TC2 have](https://www.newtek.com/tricaster/2-elite/tech-specs/) LiveStreaming and SRT support. It seems that it can publish on youtube facebook, etc. It likely means it can publish to Ant Media Server with RTMP. 


Anyway, Ant Media Server can ingest RTMP and SRT that is published by any encoder.  No need to purchase another component if your TC2 already supports RTMP or SRT. 

FYI
